SwiftyAttributes/Sources/common/AttributeContainer+SwiftyAttributes.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  AttributeContainer+SwiftyAttributes.swift
+//  SwiftyAttributes
+//
+//  Created by Roman Podymov on 29/04/23.
+//  Copyright © 2023 Roman Podymov. All rights reserved.
+//
+
+import Foundation
+
+#if canImport(UIKit)
+@available(iOS 15.0, tvOS 15.0, watchOS 8.0, *)
+public extension AttributeContainer {
+    mutating func set(attribute: Attribute) {
```

Comment:
not necessary in this PR, but would be neat to also add a custom subscript to `set` and `get` the attributes

Reply:
Hello @eddiekaiger 
Do you mean something like
```
public extension AttributeContainer {
    subscript(attribute: Attribute) -> Any? {
        get { /* */ }
        set { /* */ }
    }
}
```
?

Reply:
yep!